cpp/src/neighbors/detail/vamana/greedy_search.cuh

```diff
@@ -64,6 +65,27 @@ __forceinline__ __device__ void sort_visited(
 
 namespace {
 
+template <typename T,
+          typename accT,
+          typename IdxT     = uint32_t,
+          typename Accessor = raft::host_device_accessor<std::experimental::default_accessor<T>,
```

Comment:
The Accessor template parameter doesn't seem to be used within the kernel. Can we remove it?

Reply:
Removed